numpy/random/pcg64.pyx

```diff
@@ -271,23 +271,20 @@ cdef class PCG64:
         golden number.
         """
         step = 0x9e3779b97f4a7c15f39cc0605cedc834
-        step *= int(iter)
-        divisor = step // 2**128
-        step -= 2**128 * divisor
-        self.advance(step)
```

Comment:
Did you mean to change the meaning here? Does `advance` already handle the `% 2**64`?

Reply:
Yes. All of the advance calls use wrap_int to ensure that the step is in bounds for the generator.  This was effectively calling wrap_int twice.

Reply:
I better triple check this.

Reply:
`advance` has a call to `wrap_int(delta, 128)` which truncates this value to be inbounds for a 2**128 state size, so that this call is unnecessary.